src/index.ts

```diff
@@ -10,43 +10,101 @@ import {
+  // REVISION - Change the variable to your renamed "AptosWalletName" variable. (Ex. "PetraWalletName")
+  readonly name = AptosWalletName; 
+
+  // REVISION - Include the link to create an account using your wallet or your primary website. 
+  // (Ex. https://chromewebstore.google.com/detail/petra-aptos-wallet/ejjladinnckdgjemekebdpeokbikhfci?hl=en)
+  readonly url = "";
 
-  // An optional property for wallets which may have different wallet name with window property name.
-  // such as window.aptosWallet and wallet name is Aptos.
-  // If your wallet name prop is different than the window property name use the window property name here and comment out line 37
+  /**
+   * REVISION - Set the icon to be a base64 encoding of your Wallet's logo.
+   * 
+   * The icon data must be of the format:
+   * 1. "data:image/"
+   * 2. The icon's file extension, which must be one of:
+   *    - "svg+xml"
+   *    - "webp"
+   *    - "png"
+   *    - "gif"
+   * 3. ";base64,"
+   * 4. The base64 encoding of the image file.
+   * 
+   * See the current value of icon for an example of this format.
+   */ 
+  readonly icon = "";
 
-  // readonly providerName = "aptosWallet";
+  // REVISION - Optionally set providerName to be the actual property name for "AptosWindow.aptos" if it is different than your AptosWalletName. 
+  // Ex. If your WalletName was "AptosWallet", but your property on "AptosWindow" was still ".aptos", you could set providerName to "aptos".
+  // readonly providerName = "aptos";
```

Comment:
lets use a different name and not `aptos` we dont want to "encourage" them to use aptos in any way